messages/pt.json

```
@@ -36,6 +36,7 @@
   "store/buy-button.add-to-cart": "Adicionar ao carrinho",
   "store/buybutton.buy-success": "Item adicionado ao seu carrinho",
   "store/buybutton.add-failure": "Algo deu errado e o item não pode ser adicionado ao seu carrinho!",
+  "store/buybutton.buy-offline-success": "Item adicionado offline ao seu cart",
```

What do you think of?
```suggestion
  "store/buybutton.buy-offline-success": "Item adicionado ao seu carrinho offline",
```

---

It makes more sense @klzns 😃 thanks